superbuild/projects_modules/medInria.cmake

```diff
@@ -90,19 +90,6 @@ set(cmake_args
   -DBUILD_ALL_PLUGINS=OFF
   -DBUILD_COMPOSITEDATASET_PLUGIN=OFF
   -DBUILD_EXAMPLE_PLUGINS=OFF
-  -DBUILD_LEGACY/ITKDATADIFFUSIONGRADIENTLIST_PLUGIN=ON
```

Comment:
Why do you remove these lignes?

Reply:
They are not used as far as i saw, and the diffeomorphic one changed its name